tests/system/python/packages/test_multiple_assets.py

```diff
@@ -300,10 +312,10 @@ def test_add_multiple_assets_before_after_restart(self, reset_fledge, start_nort
         if not skip_verify_north_interface:
             assert old_ping_result['dataSent'] < new_ping_result['dataSent']
             _verify_egress(read_data_from_pi_web_api, pi_host, pi_admin, pi_passwd, pi_db, wait_time, retries,
-                           total_benchmark_services)
-
-    def test_multiple_assets_with_reconfig(self, reset_fledge, start_north, read_data_from_pi_web_api, skip_verify_north_interface,
-                                           fledge_url,
+                           total_benchmark_services * 2, num_assets_per_service)
```

Comment:
what is * 2

Reply:
In this test, initially total_benchmark_services is 3 but after the restart the 3 more south services are added.
So,  total_benchmark_services * 2 is 6